src/Servers/Kestrel/Core/src/Internal/Http/Http1MessageBody.cs

```diff
@@ -1,223 +1,18 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
-using System.Buffers;
-using System.Diagnostics;
-using System.IO;
-using System.IO.Pipelines;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Connections;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure;
-
 namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http
 {
     public abstract class Http1MessageBody : MessageBody
     {
-        private readonly Http1Connection _context;
-
-        private volatile bool _canceled;
-        private Task _pumpTask;
+        protected readonly Http1Connection _context;
```

Comment:
This now only contains the "For" method as the other message bodies are in different classes. I may restore them here if the diff is too nasty.

Reply:
You might as well delete the class then. You can leave a static class for the For method, but otherwise all it's doing is setting the _context field in the ctor.

Reply:
Never mind. I see you put more shared logic here. 👍 Carry on.